### ISSUE TITLE
Compilation guide (Android from Windows)

### ISSUE THREAD

**Issue by brentglover** [libretro/docs#113]
The old page points to the docs here but I cannot see the Android from Windows Compilation Guide.

Brent

**Comment by thatman84** [comment]
@markkidd did you back up this guide?

**Comment by thatman84** [comment]
Oops @markwkidd

**Comment by markwkidd** [comment]
Hi @brentglover please use this link temporarily: https://github.com/libretro/RetroArch/wiki/Compilation-guide-(Android-from-Windows)/240f2283943c740ab972743b5efe9a4f6dfd56fc

**Comment by markwkidd** [comment]
@thatman84 When I last worked on the android compilation docs, the two docs were merged into one (compiling android from windows + compiling android from Linux). But what has happened to that doc? Hmm. Let me keep looking

**Comment by markwkidd** [comment]
@brentglover please disregard my prior message. Here is a temporary link to the Android compilation guide: https://github.com/markwkidd/docs/blob/8e3d34c20340370a9a254c877e7740580ff12006/docs/compilation/android.md

**Comment by brentglover** [comment]
OK Many thanks for your Help.

**Comment by markwkidd** [comment]
@brentglover glad to. I noticed that parts of that guide are in pretty rough shape.

If you see anything that needs to be corrected while you're working with it, please feel free to either submit a PR to make changes or post here again to let us know.

**Comment by brentglover** [comment]
I am running through it from the beginning and documenting.  If successfull, I can send the guide without the "check google for this bit" ;)

**Comment by markwkidd** [comment]
That would be great, thanks!

**Comment by thatman84** [comment]
I failed at following that guide. It would be great to see an enhanced version.

Did it get lost around the time the older windows compilation guides got removed by accident from your PR

**Comment by thatman84** [comment]
Any news @brentglover?

Same brent from the android split cores thread?

**Comment by brentgl** [comment]
Yes same brent from the split cores thread.  So yes I have managed to get a working system and run an online updater and a info updater.  I have documented all steps to get a valid build environment working under ubuntu though in the end.  Not an easy mission - ups the respect I have for the devs here no end in trying to perform this most basic task lol.

Just looking at the recipes folder in the libretro-super folder shows that there isnt a one size fits all approach but I will try to get something understandable out


Brent

**Comment by thatman84** [comment]
brilliant stuff. Yeah the devs really are a world away from "normal" users :)
